app/tests/src/test/resources/setup.sh

```diff
@@ -3,44 +3,29 @@
 #   External Environment Variables
 #
 #1. KUBERNETES_NAMESPACE:                   The target namespace.
-#2. SYNDESIS_TEMPLATE_TYPE:                 The template type to use (e.g. syndesis, syndesis-restricted, syndesis-ephemeral-restricted.
-#3. SYNDESIS_TEMPLATE_URL:                  The full url to the templates to use.
-#4. OPENSHIFT_TEMPLATE_FROM_WORKSPACE:      Flag to enable reading the template from the ${WORKSPACE} environment variable.
-#5. OPENSHIFT_TEMPLATES_FROM_GITHUB_COMMIT: The sha of the actual commit to use. Meant to be used for pull request validation (currently unused?).
+#2. OPENSHIFT_MASTER                        The openshift master url.
+#3. DEMO_DATA_ENABLED                       Demo data flag.
+#4. MINISHIFT_ENABLED                       Minishift flag.
 #
 #   Template Parameter Values:
-#
 #   OPENSHIFT_MASTER
 
-#Configure the SYNDESIS_TEMPLATE_TYPE
-if [ -z "${SYNDESIS_TEMPLATE_TYPE}" ]; then
-    SYNDESIS_TEMPLATE_TYPE="syndesis-ci"
-fi
-
 if [ -z "${KUBERNETES_NAMESPACE}" ]; then
     KUBERNETES_NAMESPACE="$(oc project -q)"
 fi
-
-#Configure the SYNDESIS_TEMPLATE_URL
-if [ -n "${SYNDESIS_TEMPLATE_URL}" ]; then
-    true
-elif [ -n "${OPENSHIFT_TEMPLATE_FROM_WORKSPACE}" ]; then
-    SYNDESIS_TEMPLATE_URL="${WORKSPACE}/${SYNDESIS_TEMPLATE_TYPE}.yml"
-elif [ -n "${OPENSHIFT_TEMPLATES_FROM_GITHUB_COMMIT}" ]; then
-    SYNDESIS_TEMPLATE_URL="https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/${OPENSHIFT_TEMPLATES_GIT_COMMIT}/${SYNDESIS_TEMPLATE_TYPE}.yml"
-else
-    SYNDESIS_TEMPLATE_URL="https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/master/${SYNDESIS_TEMPLATE_TYPE}.yml"
-fi
-
-#Configure OPENSHIFT_MASTER
 if [ -z "${OPENSHIFT_MASTER}" ]; then
     OPENSHIFT_MASTER="$(oc whoami --show-server)"
 fi
-
 if [ -z "${DEMO_DATA_ENABLED}" ]; then
     DEMO_DATA_ENABLED=false
 fi
 
+SYNDESIS_TEMPLATE_TYPE="syndesis-restricted"
+SYNDESIS_TEMPLATE_URL="${WORKSPACE}/app/deploy/${SYNDESIS_TEMPLATE_TYPE}.yml"
```

Comment:
You could export WORKSPACE to $(appdir) in the system-test script and the use just "${WORKSPACE}/deploy/..." here (without 'app'). but yeah, nit picking on high level.